app/src/main/java/org/mozilla/focus/search/MultiselectSearchEngineListPreference.java

```diff
@@ -48,4 +49,36 @@ protected void updateDefaultItem(CompoundButton defaultButton) {
         }
         return engineIdSet;
     }
+
+    // Whenever an engine is checked or unchecked, we adapt the MenuItem state (enable or disable) given in param
+    // Though some conditions are added just for optimisation
+    public void bindEnginesCheckboxesTo(final MenuItem menuDeleteItem, final int alpha_default, final int alpha_greyed) {
```

Comment:
Nit: I think the "To" naming convention is more of a Swift/iOS pattern, so I'd remove the "to", but up to you.

Reply:
Oh I never tried Swift, I wanted just to make it clear that we are binding the engines to the trash menuItem in param.

What about the name `bindEnginesCheckboxesToMenuItem` ?

Reply:
That sounds good to me! Though to be specific, probably Engine singular, since each checkbox has one engine.